### ISSUE TITLE
[BUG] `none-ls` broken

### ISSUE THREAD

**Issue by gador** [nix-community/nixvim#2125]
| Field        | Description |
|--------------|-------------|
| Plugin       | `none-ls`  | 
| Nixpkgs      | `5629520edecb69630a3f4d17d3d33fc96c13f6fe`  (current master)| 
| Home Manager | `unstable` | 

- [x] I have read the [FAQ](https://nix-community.github.io/nixvim/user-guide/faq.html) and my bug is not listed there.

## Description

After an update of my flake, the following error appears:
```
       … while calling anonymous lambda

         at /nix/store/g50vm3mi7j7ysaimxryw0hv7f95x0lrf-source/lib/modules.nix:648:14:

          647|           optionDecls = filter
          648|             (m: m.options?_type
             |              ^
          649|                 && (m.options._type == "option"

       … from call site

         at /nix/store/bbww7acvbh82z5air318r5iqn2cy1rcb-source/plugins/none-ls/_mk-source-plugin.nix:58:12:

           57|         }
           58|         // lib.optionalAttrs (pkg != null) { default = pkg; }
             |            ^
           59|       );

       … while calling 'optionalAttrs'

         at /nix/store/g50vm3mi7j7ysaimxryw0hv7f95x0lrf-source/lib/attrsets.nix:1352:5:

         1351|     cond:
         1352|     as:
             |     ^
         1353|     if cond then as else {};

       error: attribute 'prisma' missing

       at /nix/store/bbww7acvbh82z5air318r5iqn2cy1rcb-source/plugins/none-ls/packages.nix:158:23:

          157|       phpcs = pkgs.phpPackages.php-codesniffer;
          158|       prisma_format = pkgs.nodePackages.prisma;
             |                       ^
          159|       ptop = pkgs.fpc;
```

## Minimal, Reproducible Example (MRE)

```nix
programs.nixvim = {
    enable = true;
};
```

**Comment by msanft** [comment]
Experiencing the same

**Comment by MattSturgeon** [comment]
Should've been fixed by #2128
